src/components/AsyncSearchableSelect.tsx

```
@@ -85,73 +87,175 @@ const AsyncSearchableSelect = <
             if (hasError) {
-              return '!border !border-danger-500';
+              return twMerge( '!border !ring-danger-500 !ring-1', userControlStyles );
+            }
+
+            return twMerge( '!border-gray-200', userControlStyles );
+          },
+        }}
+        menuPortalTarget={props.menuPortalTarget}
+        menuPosition={props.menuPosition}
+        styles={{
+          ...(props?.styles ? (props.styles as any) : {}),
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)